src/calendars/NepaliLocaleNp.js

```diff
@@ -0,0 +1,33 @@
+const nepali_np = {
+    name: "nepali_np",
```

Comment:
@dragos-dobre this is for @purusap to update, I do not know proper names in nepali.